schema/test.json

```diff
@@ -6753,13 +6753,13 @@
 
     },
     {
-        "path"     : "/part[0,15]/kit[0,15]/subpars/PAmpVelocityScaleFunction",
+        "path"     : "/part[0,15]/kit[0,15]/subpars/AmpVelocityScaleFunction",
         "shortname": "sense",
-        "name"     : "PAmpVelocityScaleFunction",
+        "name"     : "AmpVelocityScaleFunction",
         "tooltip"  : "Amplitude Velocity Sensing function",
-        "type"     : "i",
-        "range"    : [0,127],
-        "default"  : "90"
+        "type"     : "f",
+        "range"    : [0,100.0],
+        "default"  : "50"
```

Comment:
This doesn't match the default value from the zynaddsubfx commit. Is this difference wanted?

Reply:
It was really silly for me to miss that 😅 thanks for the review!